src/styles/menu-slider.js

```diff
@@ -3,13 +3,19 @@ import { css } from 'lit-element';
 const menuButtonWidth = css`42px`;
 
 export default css`
+
+  @media (max-width: 500px) {
```

Comment:
In general, I would try to avoid having web components take into account layout outside of themselves (the window size in this case) and instead use something like a ResizeObserver to change its layout based on its available viewport.

Reply:
yah, I wish I didn't have to do neither the media query nor the resizeObserver. I would just like CSS width to cascade according to whatever its parent is doing.
Right now, the CSS is a lil brittle... I'll retry to control it on the parent container side again, if I can, I'll yank.

Reply:
ok ok, I just took a quick look, and it looks like it's an easy enough change on the component level but, the we have more animated transitions around this menu's wrapper in Petabox, that I don't
 want to rattle right before beta QA.

I do have a branch for it here: https://github.com/internetarchive/iaux-menu-slider/tree/get-rid-of-media-query

and I will definitely circle back once the dust settles..

Reply:
I made a ticket - https://github.com/internetarchive/iaux-menu-slider/issues/9
The reduction of responsibility is an important one that we need to be vigilant about.